.github/workflows/main.yml

```diff
@@ -68,12 +76,12 @@ jobs:
             ~/.cache/bazel-repo
           key: v3-bazel-repo-cache-${{ hashFiles('.bazelversion', 'examples/WORKSPACE', 'repositories/**', 'requirements_lock.txt', 'WORKSPACE') }}
```

Comment:
Maybe we would want a separate cache per toolchain? Wasn't sure about that. In that case we'd have to include `${{ matrix.toolchain }}` in the `key` line.

Reply:
I don't think this is necessary, as those files are relevant for the repository cache, only.